.github/workflows/generate_sitemap_new.yml

```
@@ -0,0 +1,41 @@
+name: Generate and Upload Combined Sitemap
+
+on:
+  workflow_dispatch:
+
+jobs:
+  generate_combined_sitemap:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: "20"
+
+      - name: Install dependencies
+        shell: bash
+        run: npm install yargs xml2js
+
+      - name: Download sitemaps
+        shell: bash
+        run: |
+          mkdir -p temp
+          curl -o temp/staging_sitemap.xml https://staging.deriv.com/sitemap.xml
+          curl -o temp/academy_sitemap.xml https://academy-v2.webflow.io/sitemap.xml
+          curl -o temp/robots.txt https://staging.deriv.com/robots.txt
```

```suggestion
          curl -o temp/staging_sitemap.xml https://staging.deriv.com/sitemap.xml || exit 1
          curl -o temp/academy_sitemap.xml https://academy-v2.webflow.io/sitemap.xml || exit 1
          curl -o temp/robots.txt https://staging.deriv.com/robots.txt || exit 1
```

just for better error handling in case the downloading fails